modules/virt-about-using-virtual-gpus.adoc

```diff
@@ -0,0 +1,104 @@
+
+[NOTE]
+====
+Refer to your hardware vendor's documentation for functionality and support details.
+====
+
+Mediated device:: A physical device that is divided into one or more virtual devices. A vGPU is a type of mediated device (mdev); the performance of the physical GPU is divided among the virtual devices. You can assign mediated devices to one or more virtual machines (VMs), though some GPUs do not support multiple guests.
+
+[id="configuration-overview_{context}"]
+== Configuration overview
+
+To configure mediated devices in the `HyperConverged` CR, an administrator must:
+
+* List the types of mediated devices that you want {VirtProductName} to create in the `spec.mediatedDevicesConfiguration` stanza.
+
+* Expose the mediated devices to the cluster by listing their details in the `spec.permittedHostDevices` stanza.
+
+Optionally, you can override the global configuration for a specific node or group of nodes by including the `spec.mediatedDevicesConfiguration.nodeMediatedDeviceTypes` stanza. This stanza must contain its own `mediatedDevicesTypes` list.
+
+[id="how-vgpus-are-assigned-to-nodes_{context}"]
+=== How vGPUs are assigned to nodes
+
+For each physical device, {VirtProductName} configures:
+
+* A single mdev type.
+* The maximum number of instances of the selected mdev type.
+
+The cluster architecture affects how devices are created and assigned to nodes.
+
+Large cluster with multiple cards per node:: On nodes with multiple cards that can support similar vGPU types, the relevant device types are created in a round-robin manner.
+For example:
++
+[source,yaml]
+----
+...
+mediatedDevicesConfiguration:
+  mediatedDevicesTypes:
+  - nvidia-222
+  - nvidia-228
+  - nvidia-105
+  - nvidia-108
+...
+----
++
+In this scenario, each node has two cards, both of which support the following vGPU types:
++
+[source,text]
+----
+nvidia-105
+...
+nvidia-108
+nvidia-217
+nvidia-299
+...
+----
++
+On each node, {VirtProductName} creates:
+
+* 16 vGPUs of type nvidia-105 on the first card.
+* 2 vGPUs of type nvidia-108 on the second card.
+
+One node has a single card that supports more than one requested vGPU type:: {VirtProductName} uses the supported type that comes first on the `mediatedDevicesTypes` list.
++
+For example, a node's card supports `nvidia-223` and `nvidia-224`. The following `mediatedDevicesTypes` list is configured:
++
+[source,yaml]
+----
+...
+mediatedDevicesConfiguration:
+  mediatedDevicesTypes:
+  - nvidia-22
+  - nvidia-223
+  - nvidia-224
+...
+----
++
+In this example, {VirtProductName} uses the `nvidia-223` type.
+
+[id="about-changing-removing-mediated-devices_{context}"]
+== About changing and removing mediated devices
+
+{VirtProductName} updates the cluster's mediated device configuration if:
+
+* An administrator edits the `HyperConverged` CR and changes the contents of the `mediatedDevicesTypes` stanza.
```

Comment:
suggest/ If you can address the user directly as "you", the following three bullets can be slightly condensed:
- you edit the ...
- you change...
- you remove...

Reply:
I had that at first and then changed it. 😆 I will change it back!